### DISCUSSION TITLE
Multiline Query

### DISCUSSION THREAD

**Discussion by MichalisDBA** [osalvador/ReplicaDB#91] [Q&A] [answered]
Hello.

How we can write a multiline query inside the config file?

```
source.query=SELECT * FROM TABLE
WHERE X=1 AND Y=2
ORDER BY X ASC
```

Something like that.

**Comment by osalvador** [comment, marked as answer]
Hi @MichalisDBA 

Just put a backslash at end-of-line: 

```properties
source.query=SELECT * FROM TABLE \
WHERE X=1 AND Y=2 \
ORDER BY X ASC
```

https://docs.oracle.com/javase/6/docs/api/java/util/Properties.html

**Reply by MichalisDBA** [reply]
Thanks. I was stuck with my bashing technique trying some << EOF etc.
